docs/tutorial/03-fungible-tokens.mdx

```diff
@@ -512,7 +512,7 @@ Capabilities allow us to accomplish this safely.
 
 ---
 
-Another important feature in Cadence is its utilization of **Capability Security.**
+Another important feature in Cadence is its utilization of [**Capability Security.**](/cadence/language/capability-based-access-control)
 This feature ensures that, while the withdraw function is declared public on the resource,
 no one except the intended user and those they approve of can withdraw tokens from their vault.
```

Comment:
Removes unneeded comma

_This feature ensures that while the withdraw function is declared public on the resource,
no one except the intended user and those they approve of can withdraw tokens from their vault._